go.mod

```diff
@@ -273,7 +273,7 @@ replace (
 	github.com/gogo/protobuf => github.com/gravitational/protobuf v1.3.2-0.20201123192827-2b9fcfaffcbf
 	github.com/gravitational/teleport/api => ./api
 	github.com/julienschmidt/httprouter => github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5
-	github.com/keys-pub/go-libfido2 => github.com/gravitational/go-libfido2 v1.5.3-0.20220412221142-6da2a1c840d1
+	github.com/keys-pub/go-libfido2 => ../go-libfido2
```

Comment:
Do we actually have a go-libfido2 directory or is this some magic syntax?

Reply:
No, we need https://github.com/gravitational/go-libfido2/pull/4 to merge. This is my local trickery, I'll amend once the prereq PR lands.